src/averell/utils.py

```diff
@@ -262,6 +262,10 @@ def write_json(poem_dict, filename):
     :param filename: string
         JSON filename that will be written with the poem data
     """
+    file_path = Path(filename)
+    if os.path.exists(f"{filename}.json"):
+        file_count = len(list(file_path.parent.glob(f"{file_path.stem}*")))
```

Comment:
Could we add the extension?
```suggestion
        file_count = len(list(file_path.parent.glob(f"{file_path.stem}*.json")))
```

Reply:
Sure, just 80 characters!